### PR TITLE
Adds uninstall_all_cmake_projects.sh script

### PR DESCRIPTION
**Pull request by luis-pereira**
It basically runs 'sudo xargs rm < install_manifest.txt' in each build
directory.

### CODE REVIEW

File: uninstall_all_cmake_projects.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	lxqt-sudo \
+	compton-conf \
+	obconf-qt"
+
+for d in ${CMAKE_REPOS}
+do
+    INSTALL_MANIFEST_FULL_PATH="${d}/build/${INSTALL_MANIFEST}"
+    if [ -f "${INSTALL_MANIFEST_FULL_PATH}" ]
+    then
+        sudo xargs rm < "${INSTALL_MANIFEST_FULL_PATH}"
```
Comment (palinek) [inline uninstall_all_cmake_projects.sh]: IMO we shouldn't force running as root. It's not needed if the instalation paths are accessible by current user. If not, the user should elevate his/her rights on his own.

Comment (luis-pereira) [inline uninstall_all_cmake_projects.sh]: @palinek Agreed as the script doesn't do anything else that could be harmful

**Comment by palinek** [comment]
@luis-pereira the @6a9a406 is just a suggestion... feel free to squash, remove or whatever if you think it's not worth...

**Comment by palinek** [comment]
@luis-pereira can you have a look on branch [build_enahnce](https://github.com/lxde/lxqt/compare/build_enahnce). Is it worth submitting?

**Pull request merged**